e2e/visual_regression/start/start_page.test.ts

```diff
@@ -0,0 +1,23 @@
+import {
+  loginAsVisualRegressionUser2,
+  webuiEndpoint,
+} from '../../utils/test-util';
+import { expect, test } from '@playwright/test';
+
+test.describe('Start page Visual Regression Test', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.setViewportSize({
+      width: 1500,
+      height: 1000,
+    });
+    await loginAsVisualRegressionUser2(page);
+  });
```

Comment:
we don't need to have to log in and go to the data page for every test. Can we leave it outside of test.describe?



```suggestion
  test.beforeEach(async ({ page }) => {
    await page.setViewportSize({
      width: 1500,
      height: 1000,
    });
    await loginAsVisualRegressionUser2(page);
  });
test.describe('Start page Visual Regression Test', () => {

```